src/API/OnboardingPlugins.php

```diff
@@ -233,6 +249,8 @@ public function install_plugin( $request ) {
 			);
 			wc_admin_record_tracks_event( 'install_plugin_error', $properties );
 
+			$this->create_install_plugin_error_inbox_notification_for_jetpack_installs( $slug );
```

Comment:
This works well for adding the note, but we'll probably need a way to remove the note as well if they install on their own.

We could check on every load if these plugins exist in active plugins, but this is a little brute force.  The `activate_PLUGINNAME` would work if we want to remove this note only after activation.

Reply:
Done 😅 

https://media.giphy.com/media/R32vNOs0qpIqI/giphy.gif

Reply:
Sorry, one more thing I missed on my last review- I think we'll want to call `create_install_plugin_error_inbox_notification_for_jetpack_installs()` on activation fail as well.

Reply:
Done - https://github.com/woocommerce/woocommerce-admin/pull/3888/files#diff-f09c46da6591739c273804943f0ce4b6R311

Reply:
This is done